starwars/src/StarwarsList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Character from './Characters'
+
+const StarWarsList = props => {
+    return (
+
+        <ul>
+            {props.starwarsChars.map((char, i) => {
+                return (
+
+                    <Character starwarsChars={props.starwarsChars} index={i} />
```

Comment:
since you're already mapping the props.starwarsChars here you can just pass the char instead of props.starwarsChars to Character component